frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -365,6 +365,12 @@
             onSuccess: function(response) {
                 var details = response.data;
                 vm.phaseSplits = details;
+                for(var i=0; i<details.length; i++) {
+                    if (details[i].visibility !== 3) {
+                        vm.phaseSplits[i].showPrivate = true;
```

Comment:
I am not sure what this line is doing. Can you please tell what it will exactly do?

Reply:
It is used to decide when to show `Private` in the challenge host leaderboard. Please refer here: https://github.com/Cloud-CV/EvalAI/pull/1804/files#diff-c09876db1a2167c9371dbcccf9a69aebR27

Reply:
Do we really have to store it as a string `Private`. Can we please not show using `ng-if` which will be much neater.